plugins/aja/CMakeLists.txt

```diff
@@ -47,30 +42,24 @@ target_sources(
           audio-repack.hpp)
 
 target_link_libraries(aja PRIVATE OBS::libobs AJA::LibAJANTV2)
+target_compile_features(aja PRIVATE cxx_std_17)
+
+if(OS_WINDOWS)
+  configure_file(cmake/windows/obs-module.rc.in win-aja.rc)
+  target_sources(aja PRIVATE win-aja.rc)
+
+  target_compile_options(aja PRIVATE /wd4996)
 
-if(OS_MACOS)
+  target_link_libraries(aja PRIVATE ws2_32.lib setupapi.lib Winmm.lib netapi32.lib Shlwapi.lib)
+  target_link_options(aja PRIVATE /IGNORE:4099)
+
+elseif(OS_MACOS)
   find_library(IOKIT IOKit)
   find_library(COREFOUNDATION CoreFoundation)
   find_library(APPKIT AppKit)
 
   target_link_libraries(aja PRIVATE ${IOKIT} ${COREFOUNDATION} ${APPKIT})
-elseif(OS_WINDOWS)
-  set(MODULE_DESCRIPTION "OBS AJA Windows module")
-  configure_file(${CMAKE_SOURCE_DIR}/cmake/bundle/windows/obs-module.rc.in
-                 win-aja.rc)
-
-  target_sources(aja PRIVATE win-aja.rc)
-
-  target_compile_options(aja PRIVATE /wd4996)
-  target_link_libraries(aja PRIVATE ws2_32.lib setupapi.lib Winmm.lib
-                                    netapi32.lib Shlwapi.lib)
-  target_link_options(aja PRIVATE "LINKER:/IGNORE:4099")
-endif()
-
-if(NOT MSVC)
   target_compile_options(aja PRIVATE -Wno-error=deprecated-declarations)
 endif()
```

Comment:
I made it `if(NOT MSVC)` to cover Clang and GCC. But here the overhaul makes it covers only macOS.

Reply:
> Yeah for some reason I never needed it on Linux.. 🤔

Did you ever build OBS with AJA support on Linux ? Because the only case where you don't need it it's when it is disabled.